### PR TITLE
Add Basic Fire Description

### PR DESCRIPTION
**Pull request by IFRFSX**

### CODE REVIEW

**Comment by SmallJoker** [comment]
Why? It's neither listed in the creative inventory, nor is it obtainable by digging.

**Comment by paramat** [comment]
However it can be 'given' to a player, so a description seems of some slight usefulness.

File: mods/fire/init.lua [hunk 1]
```diff
@@ -43,6 +43,7 @@ end
 -- Flame nodes
 
 minetest.register_node("fire:basic_flame", {
+	description = S("Basic Flame"),
```
Comment (paramat) [inline mods/fire/init.lua]: I think just 'Flame' is a better description.
Perhaps both nodes should be described as 'Fire' instead?

Comment (IFRFSX) [reply mods/fire/init.lua]: The characteristics of the two kinds of flames are different.for basic flame, I think it's OK to change it to "fire".

**Comment by IFRFSX** [comment]
One suggestion: you can add basic fire to creative inventory! In this way, the server administrator can easily place the fire.

**Comment by paramat** [comment]
But then creative players could easily place it too, i odubt server admin would acceot that.

**Comment by IFRFSX** [comment]
@paramat 
ok.

File: mods/fire/init.lua [hunk 2]
```diff
@@ -35,6 +35,7 @@ local function flood_flame(pos, _, newnode)
 end
 
 -- Flame nodes
+
```
Comment (paramat) [inline mods/fire/init.lua:38]: This empty line is inconsistent formatting.

**Comment by paramat** [comment]
The description of fire:permanent_flame is still inconsistent with the decription of fire:basic_flame.

File: mods/fire/locale/fire.zh_TW.tr [hunk 3]
```diff
@@ -1,3 +1,4 @@
 # textdomain: fire
-Permanent Flame=永久火焰
+Permanent Fire=永久火焰
 Flint and Steel=火石和鋼
+Fire = 火焰
```
Comment (MoNTE48) [inline mods/fire/locale/fire.zh_TW.tr:4]: Pls, remove spaces
-Fire = 火焰
+Fire=火焰

File: mods/fire/init.lua (hunk 2 above)
Comment (paramat) [reply mods/fire/init.lua]: Fixed.

File: mods/fire/locale/fire.zh_TW.tr (hunk 3 above)
Comment (paramat) [reply mods/fire/locale/fire.zh_TW.tr]: Done.

**Comment by paramat** [comment]
Seems good now.

**Pull request merged**